packages/skeleton-svelte/src/components/app-bar/anatomy/headline.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts" module>
+	import type { HTMLAttributes } from '@/internal/html-attributes';
+	import type { PropsWithElement } from '@/internal/props-with-element';
+
+	export interface AppBarHeadlineProps extends PropsWithElement<'div'>, HTMLAttributes<'div'> {
+		children: Snippet;
+	}
+</script>
+
+<script lang="ts">
+	import { classesAppBar } from '@skeletonlabs/skeleton-common';
+	import { mergeProps } from '@zag-js/svelte';
+	import type { Snippet } from 'svelte';
+
+	const { element, children, ...rest }: AppBarHeadlineProps = $props();
```

Comment:
```suggestion
    const props: AppBarHeadlineProps = $props();
	const { element, children, ...rest } = $derived(props);
```
It's how other components also handle props, just a convention, please do this for all other components as well.